platform-core-module/src/main/java/com/softicar/platform/core/module/start/page/StartPageDiv.java

```diff
@@ -39,7 +39,7 @@ public StartPageDiv() {
 
 	private void addPendingSystemEventsSection(Collection<IDomElement> sections) {
 
-		if (CorePermissions.SYSTEM_ADMINISTRATION.test(CurrentUser.get())) {
+		if (CorePermissions.ADMINISTRATION.test(AGCoreModuleInstance.getInstance(), CurrentUser.get())) {
```

Comment:
```suggestion
		if (CurrentUser.get().isAdministrator()) {
```
please look for similar places to simplify